site/content/docs/5.2/components/list-group.md

```diff
@@ -156,7 +158,9 @@ Add `.list-group-horizontal` to change the layout of list group items from verti
 
 ## Contextual classes
 
-Use contextual classes to style list items with a stateful background and color.
+<!-- Boosted mod: Add situational icons instead of changing the bg-color -->
+Use contextual classes to style list items with a stateful icon.
+<!-- End mod -->
 
 {{< example >}}
```

Comment:
It could be useful to show in this example the selected state on one of the item with icon to show how it looks.